### PR TITLE
Adding types for srcds-rcon.

### PR DESCRIPTION
**Pull request by jdeurt**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/srcds-rcon/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+    /**
+     * Send a command to the server.
+     * @param text - The command to send.
+     * @param timeout - The time to wait before throwing an error.
+     */
+    command(text: string, timeout?: number): Promise<string>;
+}
+
+export default Rcon;
```
Comment (amcasey) [inline types/srcds-rcon/index.d.ts]: Should this be `export = Rcon;`?  From the sample usage, it looks like `require('srcds-rcon')` is supposed to produce a callable object.

File: types/srcds-rcon/tslint.json [hunk 2]
```diff
@@ -0,0 +1,80 @@
+{
+    "extends": "dtslint/dt.json",
+
+    "rules": {
```
Comment (amcasey) [inline types/srcds-rcon/tslint.json]: Why disable the lint rules?

**Review by amcasey — CHANGES_REQUESTED** [review]

File: types/srcds-rcon/index.d.ts (hunk 1 above)
Comment (jdeurt) [reply types/srcds-rcon/index.d.ts]: Yea you're right. Let me fix that real quick.

File: types/srcds-rcon/srcds-rcon-tests.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import Rcon from "srcds-rcon";
```
Comment (amcasey) [inline types/srcds-rcon/srcds-rcon-tests.ts]: If you're going to us an ES6 import (vs `import Rcon = require("srcds-rcon");`), you'll need to add either `allowSyntheticDefaultImports` or `esModuleInterop` to your tsconfig.json.

**Pull request closed without merging**